external/anomaly/anomaly_classification/openvino.py

```diff
@@ -133,51 +133,42 @@ def optimize(
         with tempfile.TemporaryDirectory() as tempdir:
             xml_path = os.path.join(tempdir, "model.xml")
             bin_path = os.path.join(tempdir, "model.bin")
-            with open(xml_path, "wb") as xml_file:
-                xml_file.write(self.task_environment.model.get_data("openvino.xml"))
-            with open(bin_path, "wb") as bin_file:
-                bin_file.write(self.task_environment.model.get_data("openvino.bin"))
```

Comment:
tbh, I'd prefer the old approach. When reading, I find it flowing better.

Reply:
I prefer it as well but I was trying to resolve `too many locals` issue. But I think it would be better to disable it as it comes at the expense of readability.

Reply:
Since we have the below 4 times in the code
```
with open(xml_path, "wb") as xml_file:
                xml_file.write(self.task_environment.model.get_data("openvino.xml"))
```

We could then maybe define a function in `optimize` method such as:
```
def __save_weights(path: str, data: byte?) -> None:
    with open(path, "wb") as file:
        file.write(data)
```